torchbenchmark/models/ADDING_MODELS.md

```diff
@@ -81,13 +81,11 @@ Important: be deliberate about support for cpu/gpu and jit/no-jit.  In the case
 your model is instantiated in an unsupported configuration, the convention is to return
 a model object from \_\_init\_\_ but raise NotImplementedError() from all its methods.
 
-See the [BenchmarkModel API](https://github.com/pytorch/benchmark/blob/master/torchbenchmark/util/model.py) to get started.
-
-Also, an [example \_\_init\_\_.py](attention_is_all_you_need_pytorch/__init__.py) from a real model.
+See the [BenchmarkModel API](https://github.com/pytorch/benchmark/blob/master/torchbenchmark/util/model.py) to get started.  Here's an [example \_\_init\_\_.py](attention_is_all_you_need_pytorch/__init__.py) from a real model.
 
 ### `set_eval()` and `set_train()`
```

Comment:
Actually we should remove this subsection now. To make the api simpler, the train and eval state set has been moved to `__init__()` function, with the argument of `test='eval'` or `test='train'`.

Reply:
Sorry, not completely getting this - the reference to model.py seems valid and useful, as is the example linked, so why do you want to delete it?

Do you think it may be better to phrase as "Get started by extending the [BenchmarkModel]((https://github.com/pytorch/benchmark/blob/master/torchbenchmark/util/model.py) base class. This implementation for the [attention is all you need](attention_is_all_you_need_pytorch/__init__.py) model can serve as a good example of how to do so

Reply:
Sorry I commented on the incorrect line... I mean let's remove this subsection:

```suggestion
### `set_eval()` and `set_train()`
```

These two APIs are now removed from the code repo, because we find it easier to setup the test early in the init stage.